mne/viz/epochs.py

```diff
@@ -88,6 +88,11 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         and is added to the image. It is typically useful to display reaction
         times. Note that it is defined with respect to the order
         of epochs such that overlay_times[0] corresponds to epochs[0].
+    gfp : bool
+        If True, plot the Global Field Power (one figure per channel type).
+        The GFP is calculated on a single-trial basis for the image, but the
+        time series plot below shows the GFP of the average across epochs,
```

Comment:
No reason I think. I wrote the code this way without thinking, then realized that there are two possible options, and decided to just doc the one I chose. If you think it should go the other way around, I'll do it like that.

I should probably reword it though. It's not the most obvious way of saying "`data.std(ch_ax).mean(trial_ax) vs. data.mean(trial_ax).std(ch_ax)`.